docs-chef-io/content/workstation/chef-workstation-app.md

```diff
@@ -38,6 +39,24 @@ Start Chef Workstation App by running the command `chef-workstation-app`.
 
 Start `Chef Workstation App` from your Applications folder.
 
+## Managing cookbooks
```

Comment:
What does this UI let you do and what will it do? We should probably start there. Also, is there a name for this window? It looks like we have two things, the menu in the taskbar and the window UI. The menu in the taskbar says "Manage Cookbooks", but the UI window title is Chef Workstation App, which might lead to confusion if this is only to handle Cookbooks.

It might make sense to start with a heading for the window UI. 

So something like:

> ## Chef Workstation App Window

> The Chef Workstation App window lets you manage Cookbooks and Chef repositories. Open the Chef Workstation App window by selecting **Manage Cookbooks** from the Chef Workstation App menu in the menu bar (macOS) or taskbar (Windows).

Reply:
Updated as suggested. Thank you!